src/particles/elements/DipEdge.H

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2021-2022 Chad Mitchell, Axel Huebl
+ *
+ * This file is part of ImpactX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_DIPEDGE_H
+#define IMPACTX_DIPEDGE_H
+
+#include "particles/ImpactXParticleContainer.H"
+
+#include <AMReX_Extension.H>
+#include <AMReX_REAL.H>
+
+#include <cmath>
+
+
+namespace impactx
+{
+    struct DipEdge
+    {
+        using PType = ImpactXParticleContainer::ParticleType;
+
+        /** Edge focusing associated with bend entry or exit
+         *
```

Comment:
Hi @cemitch99, does it make sense to add the reference of the model you used here?
```suggestion
         *
         * This model assumes a first-order effect of nonzero gap.
         * Reference: ...
         *
```

Reply:
Yes, I would add two references:

K. L. Brown, SLAC Report No. 75, 1982.
K. Hwang and S. Y. Lee, PRAB 18, 122401 (2015).

Here we use the linear fringe field map, given to first order in g/rc (gap / radius of curvature).

Reply:
Perfect, added :)